packages/react-google-maps-api/src/components/dom/OverlayView.tsx

```diff
@@ -44,14 +70,71 @@ export interface OverlayViewProps {
   children?: ReactNode | undefined
   // required
   mapPaneName: PaneNames
-  getPixelPositionOffset?: ((offsetWidth: number, offsetHeight: number) => { x: number; y: number }) | undefined
-  bounds?: google.maps.LatLngBounds | google.maps.LatLngBoundsLiteral | undefined
-  position?: google.maps.LatLng | google.maps.LatLngLiteral | undefined
+  position: google.maps.LatLng | google.maps.LatLngLiteral
```

Comment:
Was there a reason why position went from optional to required here?

Reply:
No clue either... I basically asked the same question and instead of an answer was told to make a PR for 1 character change. I just downgraded.

Reply:
It's not just a 1 line change to fix. The problem isn't just that the ` | undefined` was taken off, it's that the existing `OverlayView` component allowed you to pass either a `position` or a `bounds`. The new `OverlayViewF` still accepts a `bounds` as optional - but doesn't do anything with it. And that's why `position` needs to be mandatory.

So, a use case - specifying a bounding box - is no longer supported. This PR probably shouldn't have been merged.

Reply:
@andrewdoro was the decision to not support `bounds` an oversight or intentional?

Reply:
I think it was an oversight, I didn't think that bounds could be used like that. The bounds logic should be added it here to make it work
https://github.com/andrewdoro/react-google-maps-api/blob/b7a9401a2ecddbf27df9ec961ee3340688a408a8/packages/react-google-maps-api/src/components/dom/Overlay.tsx#L13-L36